gdk4/src/toplevel_size.rs

```diff
@@ -0,0 +1,144 @@
+// Take a look at the license at the top of the repository in the LICENSE file.
+
+use glib::translate::*;
+
+#[derive(Clone, Copy, Debug)]
```

Comment:
This should probably not be `Copy` and `Clone` if it's supposed to be opaque and just passed around as a pointer